docs/source/explanations/structures.md

```diff
@@ -179,18 +179,21 @@ It has two parts:
 * `meta` --- This contains the names and data types of the columns and index. To
   generate this we build a dataframe with zero rows in it but the same columns
   and indexes as the original, and then serialize that with Arrow.
-* `divisions` --- This contains the index values the delineate each partition.
+* `divisions` --- This contains the index values that delineate each partition.
   We generate this in a similar way.
 
 Both of the concepts (and their names) are borrowed directly from
 dask.dataframe. They should enable any client, including in languages other than
 Python, to perform the same function.
 
-### Variable
 
 As stated above, in this context all xarray structures can be thought of as
 containers of array structures. They have no microstructure of their own, only a
 macrostructure that contains array structures.
+### Variable (xarray)
+[Variable](http://xarray.pydata.org/en/stable/user-guide/terminology.html#term-Variable) is a 
+low-leve class in xarray that describes a single array. 
```

Comment:
low level

Reply:
Thanks for the review, @clintonroy! Fix pushed.